build-test.cmd

```diff
@@ -104,6 +104,13 @@ call %__ProjectDir%\init-tools.cmd
 
 REM =========================================================================================
 REM ===
+REM === Deploy debugger tests
```

Comment:
Why don't you do these in runtest.cmd like what you for runtest.sh ?
So that tests can be run independent of the build